.golangci.yml

```diff
@@ -74,13 +91,10 @@ linters:
       rules:
         # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#blank-imports
         - name: blank-imports
-          disabled: false
```

Comment:
Aren't changes from this point to the bottom not related to `gocritic` but mostly `revive`? If so then could it be extracted to a separate PR?